scripts/scenario_07_quick_basic.lua

```diff
@@ -155,7 +158,8 @@ function init()
     player:setPosition(random(-2000, 2000), random(-2000, 2000)):setCallSign(ship_names[math.random(1,#ship_names)])
     player:setJumpDrive(true)
     player:setWarpDrive(false)
-    allowNewPlayerShips(false)
```

Comment:
Have multiple player ships breaks various small things in the scenario. As the primary ship is used for messages and a few other things.

Reply:
You are right, additional ships in Quick Basic work only as support ships. (I might adapt the scenario at another time.) For now, I removed that change to finish this pull request quickly.